optimum/exporters/onnx/model_configs.py

```diff
@@ -1052,7 +1016,7 @@ def torch_to_onnx_input_map(self) -> dict[str, str]:
         return {"x": "pixel_values"}
 
 
-@register_tasks_manager_onnx("mgp-str", *["feature-extraction", "image-to-text"])
```

Comment:
the model doesn't support image to text but rather [scene text recognition](https://huggingface.co/docs/transformers/en/model_doc/mgp-str#transformers.MgpstrForSceneTextRecognition)